instaseis/base_instaseis_db.py

```diff
@@ -34,6 +34,126 @@ class BaseInstaseisDB(with_metaclass(ABCMeta)):
     """
     Base class for all Instaseis database classes defining the user interface.
     """
+    def get_greens_seiscomp(self, epicentral_distance_degree,
+                            source_depth_in_m, origin_time=UTCDateTime(0),
+                            kind='displacement', return_obspy_stream=True,
+                            dt=None, kernelwidth=12):
```

Comment:
Do you mean if I rename this to `greens_function(..., definition="seiscomp")`. Then we could easily add other definitions/formats in the future.